lib/dialects/mssql/index.js

```diff
@@ -1,32 +1,15 @@
 'use strict';
 
 const _ = require('lodash');
-const AbstractDialect = require('../abstract');
+const { AbstractDialect } = require('../abstract');
 const ConnectionManager = require('./connection-manager');
 const Query = require('./query');
 const QueryGenerator = require('./query-generator');
 const DataTypes = require('../../data-types').mssql;
 const { MSSqlQueryInterface } = require('./query-interface');
 
 class MssqlDialect extends AbstractDialect {
-  constructor(sequelize) {
-    super();
-    this.sequelize = sequelize;
-    this.connectionManager = new ConnectionManager(this, sequelize);
-    this.queryGenerator = new QueryGenerator({
-      _dialect: this,
-      sequelize
-    });
-    this.queryInterface = new MSSqlQueryInterface(
-      sequelize,
-      this.queryGenerator
-    );
-  }
-}
-
-MssqlDialect.prototype.supports = _.merge(
-  _.cloneDeep(AbstractDialect.prototype.supports),
-  {
+  static supports2 = _.merge(_.cloneDeep(AbstractDialect.supports), {
```

Comment:
I think this can be removed right?

Reply:
🤦‍♀️

I added that to ensure both versions were deep equal, removing